packages/lesswrong/components/comments/CommentsTimelineSection.tsx

```diff
@@ -63,12 +77,25 @@ const CommentsTimelineSection = ({
   refetch?: any,
   classes: ClassesType,
 }) => {
-  const currentUser = useCurrentUser();
-  
+  const { history } = useNavigation();
+  const location = useLocation();
+  const { query } = location;
```

Comment:
Nit: With the usage here, can we just do `const {location, query} = useLocation()`